example/.env.sample

```diff
@@ -1,3 +0,0 @@
-MILLICAST_STREAM_NAME=yourStreamName
-MILLICAST_ACCOUNT_ID=yourAccountId
-MILLICAST_PUBLISH_TOKEN=yourPublishToken
```

Comment:
This file shouldn't be deleted as it's provided as an example and it's also mentioned in the README.